src/InformaticsGateway/Services/Export/ExportServiceBase.cs

```diff
@@ -145,6 +145,7 @@ private async Task OnMessageReceivedCallback(MessageReceivedEventArgs eventArgs)
             if (Interlocked.Read(ref _activeWorkers) >= Concurrency)
             {
                 _logger.ExceededMaxmimumNumberOfWorkers(ServiceName, _activeWorkers);
+                await Task.Delay(200).ConfigureAwait(false); // small delay to stop instantly dead lettering the next message.
```

Comment:
between acknowledge message and releasing the   Interlocked.Decrement call, new message can be received in a tight loop, sending them to the deadletter queue.